maintainers/maintainer-list.nix

```
@@ -3699,6 +3699,12 @@
     githubId = 362833;
     name = "Hongchang Wu";
   };
+  hoverbear = {
+    email = "operator+nix@hoverbear.org";
+    github = "hoverbear";
+    githubId = "130903";
```

```suggestion
    githubId = 130903;
```

Please put this into a separate commit with the message ``maintainers: add hoverbear``.